src/app/dataset-view/additional-components/lineage-component/lineage.component.html

```diff
@@ -1,20 +1,19 @@
-<div class="search-result-container__content" id="searchResultContainerContent">
-    <ng-template
-        [ngIf]="
-            isAvailableLineageGraph &&
-            lineageGraphNodes &&
-            lineageGraphLink.length !== 0 &&
-            lineageGraphNodes.length !== 0
-        "
-    >
+<div
+    class="search-result-container__content"
+    id="searchResultContainerContent"
```

Comment:
Could you please change camelCalse to kebab-case?
```suggestion
    id="search-result-container-content"
```
Also please check for usage in styles, tests, and ts code

Reply:
Removed this id. No longer used